packages/devtools-launchpad/bin/development-server.js

```diff
@@ -35,7 +35,13 @@ function httpGet(url, onResponse) {
 function serveRoot(req, res) {
   const tplPath = path.join(__dirname, "../index.html");
   const tplFile = fs.readFileSync(tplPath, "utf8");
-  res.send(Mustache.render(tplFile, { isDevelopment: isDevelopment() }));
+  res.send(Mustache.render(tplFile, {
+    isDevelopment: isDevelopment(),
+    pageTitle: getValue("pageTitle") || getValue("title"),
+    favicon: getValue("assets.favicon")
+      ? path.basename(getValue("assets.favicon"))
+      : "launchpad-favicon.png"
```

Comment:
Hmm not sure about this one. Perhaps tools add their favs in the same launchpad dir :)